recipe/meta.yaml

```
@@ -27,23 +33,33 @@ requirements:
     - fftw 3.3.9
   run:
     - python
+    #_openmp_mutex AND llvm-openmp needs to avoid run_exports problem
     - _openmp_mutex  # [linux]
     - llvm-openmp    # [osx]
-    - fftw # bound set through run_exports
     - {{ pin_compatible('numpy') }}
```

```suggestion
    - numpy >=1.16.6
```
numpy now has run_exports, and is backward compatible.